src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -2,6 +2,7 @@
 
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Colors[] colors = new Colors[] { Colors.RED, Colors.GREEN, Colors.BLUE, Colors.YELLOW };
+        return colors[(int) (Math.random() * 4)].toString();
```

Comment:
you can get all colors using the `values()` method
```
Color[] colors = Color.values();
```
and you can replace a number 4 with `colors.length`. It will return the number of colors from enum